pkg/build/apis/build/types.go

```diff
@@ -304,6 +304,9 @@ type BuildStatus struct {
 	// including start time, duration (in milliseconds), and the steps that
 	// occured within each stage.
 	Stages []StageInfo
+
+	// LogTail is the last few lines of the build log.  This value is only set for builds that failed.
+	LogTail *string
```

Comment:
ok.  also @csrwng suggested truncating the beginning instead of the end of the line, thoughts on that?  I tend to agree the end might contain the more useful information (vs the timestamp at the beginning).